ts/Dashboards/Plugins/DataGridComponent.ts

```diff
@@ -22,7 +22,6 @@
 import type Cell from '../Layout/Cell';
 import type DataGrid from '../../DataGrid/DataGrid';
 import type DataTable from '../../Data/DataTable';
-import type DataGridOptions from '../../DataGrid/DataGridOptions';
 import type BaseDataGridOptions from '../../DataGrid/DataGridOptions';
```

Comment:
I'm voting in favour of `DataGridOptions` instead of `BaseDataGridOptions`

Reply:
There is a name conflict, so maybe let's leave the `BaseDataGridOptions` :)

Reply:
ah, i didn't see that ;)